.github/workflows/.travis.yml

```diff
@@ -0,0 +1,42 @@
+name: ".travis.yml"
```

Comment:
Having "travis" in the file name doesn't make sense because we aren't using travis any more, and I don't think the file should be hidden either, so I would rename it to `tox.yml` (both this name field and the file itself).